sirepo/pkcli/jupyterhublogin.py

```diff
@@ -25,14 +33,54 @@ def create_user(email):
     import sirepo.sim_api.jupyterhublogin
     import sirepo.template
 
+    def maybe_create_sirepo_user(module, email, display_name):
+        u = module.unchecked_user_by_user_name(email)
+        if u:
+            # Fully registered email user
+            r = sirepo.auth_db.UserRegistration.search_by(uid=u)
+            assert r.display_name, \
+                f'uid={u} authorized AuthEmailUser record but no UserRegistration.display_name'
+            if not r.display_name == display_name:
```

Comment:
Don't change the name, just leave it. The purpose is not to ensure a name. Let the person's registration override what we do

Reply:
b5e8a8431